pkg/action/install.go

```diff
@@ -20,7 +20,7 @@ func (i *Install) Run(c *claim.Claim, creds credentials.Set, w io.Writer) error
 		return err
 	}
 
-	op, err := opFromClaim(claim.ActionInstall, c, invocImage, creds, w)
+	op, err := opFromClaim(claim.ActionInstall, false, c, invocImage, creds, w)
```

Comment:
This is more of a nit, but would it increase readibility if the `false` here passed for `stateless` was a constant?

Same for the other places where `opFromClaim` is used without directly passing the action stateless field.

Reply:
good call